ci/provisioning/post_provision_config_nodes_EL_8.sh

```diff
@@ -90,6 +66,19 @@ baseurl=${REPOSITORY_URL}repository/rocky-\$releasever-proxy/extras/\$arch/debug
 enabled=0
 gpgcheck=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-rockyofficial
+EOF
+    else
+        if [ -f /etc/yum.repos.d/daos_ci-centos8.repo ]; then
+            echo >> /etc/yum.repos.d/daos_ci-centos8.repo
+        fi
+
+        cat <<EOF >> /etc/yum.repos.d/daos_ci-centos8.repo
+[daos_ci-centos8-artifactory-debuginfo]
+name=daos_ci-centos8-artifactory-debuginfo
+baseurl=https://artifactory.dc.hpdd.intel.com/artifactory/centos-debuginfo-proxy/$releasever/$basearch/os/
```

Comment:
(style)  line over 100 characters
(lint) releasever is referenced but not assigned. [SC2154]
(lint) basearch is referenced but not assigned. [SC2154]